### PR TITLE
Remove deprecates

### LINKED ISSUE #68: Remove deprecated methods

### PR DESCRIPTION
**Pull request by Segelzwerg**
fixes #68

removed all methods and variables marked as deprecated

### CODE REVIEW

File: jpx/src/main/java/io/jenetics/jpx/GPX.java [hunk 1]
```diff
@@ -1792,82 +1721,7 @@ public static void write(final GPX gpx, final Path path) throws IOException {
...
-	/**
-	 * Writes the given {@code gpx} object (in GPX XML format) to the given
-	 * {@code output} stream.
-	 *
-	 * @param gpx the GPX object to write to the output
-	 * @param path the output path where the GPX object is written to
-	 * @throws IOException if the writing of the GPX object fails
-	 * @throws NullPointerException if one of the given arguments is {@code null}
-	 */
-	public static void write(final GPX gpx, final String path) throws IOException {
```
Comment (jenetics) [inline jpx/src/main/java/io/jenetics/jpx/GPX.java]: I didn't deprecate this write method.

**Comment by jenetics** [comment]
I can merge this PR only into 'r2.0.0' branch, which I haven't created yet. There is also no schedule for the 2.0.0 release. So it will stay open for now.

File: jpx/src/main/java/io/jenetics/jpx/GPX.java (hunk 1 above)
Comment (Segelzwerg) [reply jpx/src/main/java/io/jenetics/jpx/GPX.java]: sorry for the mishap

**Comment by Segelzwerg** [comment]
> I can merge this PR only into 'r2.0.0' branch, which I haven't created yet. There is also no schedule for the 2.0.0 release. So it will stay open for now.

no worries, if something regarding deprecation is coming up, I'm happy to help out.

**Pull request merged**